src/tests/ui/widgets/CategoryWidgetUI.test.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import { CategoryWidgetUI } from 'src/ui';
```

Comment:
We can add a `jsonconfig` file right? For relative paths from `src`

Reply:
I agree